Code/GraphMol/Kekulize.cpp

```diff
@@ -460,7 +460,12 @@ void kekulizeFused(RWMol &mol, const VECT_INT_VECT &arings,
     // number of backTracks) and we still need to backtrack
     // can't kekulize this thing
     std::ostringstream errout;
-    errout << "Can't kekulize mol " << std::endl;
+    errout << "Can't kekulize mol.";
+    errout << "  Atoms remaining:";
+    for (unsigned int i = 0; i < nats; ++i) {
+      if (dBndCands[i]) errout << " " << i;
```

Comment:
No, I think the message is good, however something like "Unkekulized Atoms" as opposed to atoms remaining might make more sense to me.

----
Brian Kelley

> On May 20, 2016, at 7:50 AM, Greg Landrum <notifications@github.com> wrote:
> 
> In Code/GraphMol/Kekulize.cpp:
> 
> > @@ -460,7 +460,12 @@ void kekulizeFused(RWMol &mol, const VECT_INT_VECT &arings,
> >      // number of backTracks) and we still need to backtrack
> >      // can't kekulize this thing
> >      std::ostringstream errout;
> > -    errout << "Can't kekulize mol " << std::endl;
> > +    errout << "Can't kekulize mol.";
> > +    errout << "  Atoms remaining:";
> > +    for (unsigned int i = 0; i < nats; ++i) {
> > +      if (dBndCands[i]) errout << " " << i;
> Do you mean you don't want any kind of message about the kekulization error or that you just don't want the atom indices but the "Can't kekulize mol." bit is ok?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>